android/src/test/java/apitests/java/mappers/CustomerInfoMapperApiTests.java

```diff
@@ -0,0 +1,13 @@
+package apitests.java.mappers;
+
+import com.revenuecat.purchases.CustomerInfo;
+import com.revenuecat.purchases.hybridcommon.mappers.CustomerInfoMapperKt;
+
+import java.util.Map;
+
+@SuppressWarnings("unused")
+class CustomerInfoMapperApiTests {
+    private void checkMap(CustomerInfo customerInfo) {
+        Map<String, Object> map = CustomerInfoMapperKt.map(customerInfo);
```

Comment:
Honestly, for all these I thought about joining the api tests into a single file... But I like better that the tests follow the original file structure of the project, even if these are not unit tests but api tests.